### ISSUE TITLE
No default whitelist set

### ISSUE THREAD

**Issue by xinsuinizhuan** [jkuri/opencv-ffmpeg-rtmp-stream#11]
avio_open2 false，and log is:
[rtmp @ 000002a52cc57440] No default whitelist set
[tcp @ 000002a52cc6e9c0] No default whitelist set
[tcp @ 000002a52cc6e9c0] Original list of addresses:
[tcp @ 000002a52cc6e9c0] Address ::1 port 1935
[tcp @ 000002a52cc6e9c0] Address 127.0.0.1 port 1935
[tcp @ 000002a52cc6e9c0] Interleaved list of addresses:
[tcp @ 000002a52cc6e9c0] Address ::1 port 1935
[tcp @ 000002a52cc6e9c0] Address 127.0.0.1 port 1935
[tcp @ 000002a52cc6e9c0] Starting connection attempt to ::1 port 1935
[tcp @ 000002a52cc6e9c0] Starting connection attempt to 127.0.0.1 port 1935
[tcp @ 000002a52cc6e9c0] Connection attempt to ::1 port 1935 failed: Error number -138 occurred
[tcp @ 000002a52cc6e9c0] Connection attempt to 127.0.0.1 port 1935 failed: Error number -138 occurred
[tcp @ 000002a52cc6e9c0] Connection to tcp://localhost:1935 failed: Error number -138 occurred
[rtmp @ 000002a52cc57440] Cannot open connection tcp://localhost:1935

**Comment by jkuri** [comment]
are you running rtmp server on rtmp://localhost:1935?

**Comment by xinsuinizhuan** [comment]
Yes.I not run server. Now it is ok! But my rtmp stream is so indistinct, how should i change the param?
![图片](https://user-images.githubusercontent.com/40679769/197346535-6d0fbfd7-b4e6-40dc-bf47-ef5a6154b4d9.png)

**Comment by jkuri** [comment]
increase resolution to a higher value, the width and height parameters.

**Comment by xinsuinizhuan** [comment]
> increase resolution to a higher value, the width and height parameters.

But the higher and width value  equare to the real image value, it to heighter to the real image value?

**Comment by jkuri** [comment]
sorry, I don't understand your question.
